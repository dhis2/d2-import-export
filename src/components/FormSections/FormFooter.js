import React from 'react'
import styles from './FormFooter.module.css'

export const FormFooter = ({ children }) => (
    <div className={styles.container}>{children}</div>
)
