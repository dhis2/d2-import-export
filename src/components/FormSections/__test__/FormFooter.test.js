import React from 'react'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import { FormFooter } from '../FormFooter'

describe('Input component - FormFooter', () => {
    it('should render correctly', () => {
        const file = shallow(<FormFooter icon={<span />} label="Form head" />)
        expect(toJson(file)).toMatchSnapshot()
    })
})
