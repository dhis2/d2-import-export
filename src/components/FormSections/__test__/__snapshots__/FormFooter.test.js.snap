// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component - FormFooter should render correctly 1`] = `
<div
  className="container"
/>
`;
